crossdock/main.go

```diff
@@ -131,18 +132,44 @@ func InitializeCollector(logger *zap.Logger) {
 
 // NewQueryService initiates the query service
 func NewQueryService(url string, logger *zap.Logger) services.QueryService {
-	cmd := exec.Command("/bin/sh", "-c", fmt.Sprintf(queryCmd,
-		"-cassandra.keyspace=jaeger -cassandra.servers=cassandra"))
-	if err := cmd.Run(); err != nil {
-		logger.Fatal("Failed to initialize query service", zap.Error(err))
-	}
+	forkCmd(
```

Comment:
Didn't want to overload the PR, only needed to diagnose query service. 

Changed other services in #247